packages/ubuntu_init/integration_test/screenshot_test.dart

```diff
@@ -106,6 +106,16 @@ Future<void> main() async {
     await tester.runApp(() => runInitApp([], theme: currentTheme));
     await tester.pumpAndSettle();
 
+    await tester.jumpToPage(InitStep.ubuntuProOnboarding.route);
+    await tester.pumpAndSettle();
+
+    final context = tester.element(find.byType(UbuntuProOnboardingPage));
+    final l10n = UbuntuProLocalizations.of(context);
+
+    final option = find.text(l10n.ubuntuProOnBoardingEnableUbuntuPro);
+    await tester.tap(option);
+    await tester.pumpAndSettle();
+
```

Comment:
We need to select the correct option first, otherwise jumping to the page won't work because its `load` method will return `false.